client/lib/analytics/utils/get-ga-gtag.ts

```diff
@@ -9,12 +10,15 @@ import { TRACKING_IDS } from '../ad-tracking/constants';
  */
 export const getGaGtag = (
 	isJetpackEnv = isJetpackCloud() || isJetpackCheckout(),
-	isAkismetEnv = isAkismetCheckout()
+	isAkismetEnv = isAkismetCheckout(),
+	isA4AEnv = isA8CForAgencies()
 ) => {
 	if ( isJetpackEnv ) {
 		return TRACKING_IDS.jetpackGoogleAnalyticsGtag;
 	} else if ( isAkismetEnv ) {
 		return TRACKING_IDS.akismetGoogleAnalyticsGtag;
+	} else if ( isA4AEnv ) {
+		return TRACKING_IDS.a4aGoogleGA4GTag;
```

Comment:
Do I need a separate tracking ID here?

Reply:
One of the tracking IDs is for Google Analytics previous to GA4. One of them starts with `UA-` and GA4 codes start with `G-` the one that you have. I am not sure if we absolutely need to have both, I'll defer to @robertsreberski on that

Reply:
If we don't need both, I'd say we should remove A4A from this `getGaGtag` function

Reply:
Ah, this one is good finding 😅 It's indeed obsolete, unused code. We can write down a follow-up clean up maintenance task for that.